### PR TITLE
Feat/domains expiration date

### PR DESCRIPTION
**Pull request by marie-j**
### Description of the Change

Replace display of domains creation date by display of domains expiration date

### Needs
- [ ]  https://github.com/ovh-ux/ovh-ui-angular/pull/334
- [ ] https://github.com/ovh-ux/ovh-utils-angular/pull/21

MANAGER-2001

### CODE REVIEW

File: client/app/domains/domains.html [hunk 1]
```diff
@@ -11,191 +11,118 @@ <h1 data-translate="domains_dashboard_title"></h1>
...
+                                data-property="name"
+                                data-type="string"
+                                data-searchable>
+                        <a class="oui-link"
+                           data-ng-bind="$row.displayName"
+                           data-ui-sref="app.domain.product({ productId: $row.displayName })">
+                        </a>
+                    </oui-column>
+                    <oui-column data-title=":: 'domains_dashboard_table_titles_expiration' | translate">
+                        <span data-ng-bind="$row.expiration | date:'mediumDate'"></span>
```
Comment (JeremyDec) [inline client/app/domains/domains.html]: You can use one-time binding no ?

File: client/app/domains/domains.html [hunk 2]
```diff
@@ -11,191 +11,118 @@ <h1 data-translate="domains_dashboard_title"></h1>
...
-                     data-pagination-server-side
-                     data-pagination-server-side-function="loadDomains"
-                     data-pagination-server-side-paginated-stuff="domains"
-                     data-pagination-server-side-table-loading="ctrlDomains.loading.domainsInfos">
-                </div>
+                <oui-datagrid data-rows-loader="ctrlDomains.loadDomains($config)"
+                              data-empty-placeholder="{{:: 'domains_dashboard_table_noresult' | translate }}"
+                              data-page-size="10"
+                              data-selectable-rows
+                              data-on-row-select="ctrlDomains.toggleDomain($row, $rows)">
```
Comment (FredericEspiau) [inline client/app/domains/domains.html]:
```suggestion
                              data-on-row-select="$ctrl.toggleDomain($row, $rows)">
```

?

File: client/app/domains/domains.routes.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ angular.module('App').config(($stateProvider) => {
     url: '/configuration/domains',
     templateUrl: 'domains/domains.html',
     controller: 'DomainsCtrl',
-    controllerAs: 'ctrlDomains',
+    controllerAs: '$ctrl',
```
Comment (FredericEspiau) [inline client/app/domains/domains.routes.js]: You could migrate to Components 😄

File: client/app/domains/domains.controller.js [hunk 4]
```diff
@@ -176,11 +117,9 @@ angular.module('App').controller(
      * Toggle the clicked domain checkbox
      * @param {string} domain name
      */
-    toggleDomain(domain) {
-      this.$scope.selectedDomains = _.xor(this.$scope.selectedDomains, [
-        domain,
-      ]);
-      this.atLeastOneSelected = this.$scope.selectedDomains.length > 0;
+    toggleDomain(domain, selectedDomains) {
+      this.selectedDomains = selectedDomains.map(({ name }) => name);
```
Comment (FredericEspiau) [inline client/app/domains/domains.controller.js]: I don't mind but some will prefer
```suggestion
       this.selectedDomains = _.map(selectedDomains, 'name');
```

**Comment by antleblanc** [comment]
@marie-j Could you please tell us why this request has been closed?

And what's about the linked Pull request as well?

- ovh-ux/ovh-ui-angular#334
- ovh-ux/ovh-utils-angular#21

Thanks:)

**Comment by antleblanc** [comment]
Seems related to: ovh-ux/ovh-manager-web#845

**Pull request closed without merging**